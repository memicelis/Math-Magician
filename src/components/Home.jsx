import React from 'react';
import styles from './Home.module.css';

const Home = () => {
  const flexContainer = {
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'center',
    justifyContents: 'center',
  };
  return (
    <div style={flexContainer}>
      <h2 className={styles.homeTitle}>Welcome to Our Page!</h2>
      <p className={styles.paragraph}>
        Lorem ipsum, dolor sit amet consectetur adipisicing elit. Id cum
        asperiores totam quaerat unde rem praesentium. Voluptate facere cum
        eveniet dicta assumenda rerum id numquam dolor, doloribus, perspiciatis
        ad at! Lorem, ipsum dolor sit amet consectetur adipisicing elit. Eos
        modi, molestiae tenetur blanditiis placeat at cum, reiciendis quidem
        quibusdam accusantium sapiente, beatae consequuntur rem minima hic enim
        cupiditate assumenda atque? Lorem ipsum dolor sit amet consectetur
        adipisicing elit. Ab ex excepturi iste in at voluptas! Officiis ratione
        nesciunt amet odio corrupti modi. Exercitationem atque, soluta
        voluptatem commodi dignissimos nostrum nihil?
      </p>
      <p className={styles.paragraph}>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam
        quaerat id tenetur magni neque! Perferendis libero voluptates laudantium
        debitis, numquam ea dolores animi quo, blanditiis quis explicabo
        consequuntur maiores beatae. Lorem ipsum dolor sit amet consectetur
        adipisicing elit. Similique voluptate ipsa a labore, rerum nisi vel,
        iure ipsum itaque quas quaerat suscipit ipsam ducimus dicta odit,
        facilis excepturi et saepe? Lorem ipsum dolor sit, amet consectetur
        adipisicing elit. Nam vitae optio similique animi numquam! Porro animi
        magni voluptates eum vero, doloremque nisi, totam repellat fuga,
        sapiente quas. Accusamus, voluptas in?
      </p>
    </div>
  );
};

export default Home;
