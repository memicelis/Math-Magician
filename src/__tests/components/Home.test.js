import React from 'react';
import Home from '../../components/Home';
import renderer from 'react-test-renderer';

it('testing Home component', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
