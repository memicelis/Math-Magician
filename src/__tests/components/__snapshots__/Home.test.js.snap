// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Home component 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flexDirection": "column",
      "justifyContents": "center",
    }
  }
>
  <h2
    className="homeTitle"
  >
    Welcome to Our Page!
  </h2>
  <p
    className="paragraph"
  >
    Lorem ipsum, dolor sit amet consectetur adipisicing elit. Id cum asperiores totam quaerat unde rem praesentium. Voluptate facere cum eveniet dicta assumenda rerum id numquam dolor, doloribus, perspiciatis ad at! Lorem, ipsum dolor sit amet consectetur adipisicing elit. Eos modi, molestiae tenetur blanditiis placeat at cum, reiciendis quidem quibusdam accusantium sapiente, beatae consequuntur rem minima hic enim cupiditate assumenda atque? Lorem ipsum dolor sit amet consectetur adipisicing elit. Ab ex excepturi iste in at voluptas! Officiis ratione nesciunt amet odio corrupti modi. Exercitationem atque, soluta voluptatem commodi dignissimos nostrum nihil?
  </p>
  <p
    className="paragraph"
  >
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam quaerat id tenetur magni neque! Perferendis libero voluptates laudantium debitis, numquam ea dolores animi quo, blanditiis quis explicabo consequuntur maiores beatae. Lorem ipsum dolor sit amet consectetur adipisicing elit. Similique voluptate ipsa a labore, rerum nisi vel, iure ipsum itaque quas quaerat suscipit ipsam ducimus dicta odit, facilis excepturi et saepe? Lorem ipsum dolor sit, amet consectetur adipisicing elit. Nam vitae optio similique animi numquam! Porro animi magni voluptates eum vero, doloremque nisi, totam repellat fuga, sapiente quas. Accusamus, voluptas in?
  </p>
</div>
`;
